### ISSUE TITLE
Ultimate service instances are not created in tests

### ISSUE THREAD

**Issue by nikitusik** [JetBrains/intellij-platform-gradle-plugin#1710]
### What happened?

When trying to create the ultimate service, an error occurs in the tests.

### Relevant log output or stack trace

```shell
@NotNull method com/intellij/database/psi/DbPsiFacade.getInstance must not return null
java.lang.IllegalStateException: @NotNull method com/intellij/database/psi/DbPsiFacade.getInstance must not return null
	at com.intellij.database.psi.DbPsiFacade.$$$reportNull$$$0(DbPsiFacade.java)
	at com.intellij.database.psi.DbPsiFacade.getInstance(DbPsiFacade.java:37)
	at com.example.ultimateservice.MySimpleTest.testUltimateServices(MySimpleTest.kt:11)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at junit.framework.TestCase.runTest(TestCase.java:177)
	at com.intellij.testFramework.UsefulTestCase.lambda$runBare$11(UsefulTestCase.java:486)
	at com.intellij.testFramework.UsefulTestCase.lambda$wrapTestRunnable$14(UsefulTestCase.java:512)
	at com.intellij.testFramework.UsefulTestCase.runTestRunnable(UsefulTestCase.java:411)
	at com.intellij.testFramework.fixtures.BasePlatformTestCase.runTestRunnable(BasePlatformTestCase.java:109)
	at com.intellij.testFramework.UsefulTestCase.defaultRunBare(UsefulTestCase.java:427)
	at com.intellij.testFramework.UsefulTestCase.lambda$runBare$12(UsefulTestCase.java:494)
	at com.intellij.testFramework.EdtTestUtil.lambda$runInEdtAndWait$3(EdtTestUtil.java:80)
	at com.intellij.openapi.application.impl.AnyThreadWriteThreadingSupport.runWriteIntentReadAction(AnyThreadWriteThreadingSupport.kt:84)
	at com.intellij.testFramework.EdtTestUtil.lambda$runInEdtAndWait$4(EdtTestUtil.java:79)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:308)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:781)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:728)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:722)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:400)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:87)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:750)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.kt:320)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:207)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:128)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:117)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:105)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:92)
```

### Steps to reproduce

see example: https://github.com/nikitusik/ultimate-service
try run `MySimpleTest`

### Gradle IntelliJ Plugin version

2.0.0-rc1

### Gradle version

8.7

### Operating System

macOS

**Comment by hsz** [comment]
Thanks for reporting, and providing a great reproducible example, Nikita!
I did compare the classpath between `1.x` and `2.0`. It turned out that by some reason, `2.0` is missing a dependency on Charts plugin.

As a workaround, you can add:

```kotlin
dependencies {
  intellijPlatform {
    // ...

    bundledPlugin("intellij.charts")
  }
}
```

**Comment by hsz** [comment]
The issue has been fixed with 109dd8008c64c128232fb664b53dbab9cb699f73 and released with `2.0.0-rc2`
